packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -743,6 +745,8 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
   final ShapeBorder? shape;
   final Color? color;
   final CapturedThemes capturedThemes;
+  final GlobalKey? menuGlobalKey;
```

Comment:
Actually, could we just make this a `Key`? I don't think we need to make it specifically a `GlobalKey` here:
```suggestion
  final Key? menuKey;
```